test/quic-openssl-docker/run_endpoint.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-set -euxo pipefail
```

Comment:
Can you explain why we don't keep any options like even `set -e`?

Reply:
set -e causes the script to abort whenever an individual command fails.  The container however requires specific exit codes to support status, so we have to be careful about what we exit with.  The others might could be reinstated, but I've not tested with them individually yet, so I was backing them out until I do

Reply:
Ah ok got it. I don't think it matters much for this script, but maybe we can consider `command || echo "Command failed"` or `command || exit 1` for future cases

Reply:
we can certainly look at that, sure.  The rule that I'm seeing in the interop-runner is that, for any given test, we need to exit with a code of either 0 1 or 127 (for unsupported tests).

I can try fix that up